src/blob.cc

```diff
@@ -17,14 +17,14 @@ Napi::FunctionReference Blob::constructor;
 void Blob::Init(const Napi::Env& env) {
   Napi::HandleScope scope(env);
 
-  Napi::Function func = DefineClass(
-      env, "Blob",
-      {InstanceMethod("byteSize", &Blob::ByteSize),
-       InstanceMethod("size", &Blob::Size),
-       InstanceMethod("rmap", &Blob::Rmap),
-       InstanceMethod("rwmap", &Blob::Rwmap),
-       InstanceMethod("wmap", &Blob::Wmap),
-       InstanceMethod("unmap", &Blob::Unmap)});
+  Napi::Function func = DefineClass(env, "Blob", {
+      InstanceMethod("byteSize", &Blob::ByteSize),
+      InstanceMethod("size", &Blob::Size),
+      InstanceMethod("rmap", &Blob::Rmap),
+      InstanceMethod("rwmap", &Blob::Rwmap),
+      InstanceMethod("wmap", &Blob::Wmap),
+      InstanceMethod("unmap", &Blob::Unmap)
+  });
```

Comment:
Do you use any script to do this code style? You can share it so that we can keep the same code style.

Reply:
Actually, I use auto formating in CLion. But it is a good idea to set up some tools to lint code. I will try it. 
As I understand you don't have CI for this repo. Am I right?

Reply:
We have coding style doc: https://github.com/intel/inference-engine-node/blob/master/doc/coding_style.md


> But it is a good idea to set up some tools to lint code. I will try it.

 +1. Thanks in advance!

Reply:
Ok, I will try this tool

Reply:
Fixed all styles after running `clang-format`. I suggest adding auto-formatting using clang-format to git hooks. For example, `clang-format` will be automatically run when we run `git add file` command